src/schemas/json/package.json

```diff
@@ -233,9 +233,9 @@
               "$ref": "#/definitions/packageExportsEntryOrFallback",
               "description": "The module path that is resolved when the module specifier matches \"name\", shadows the \"main\" field."
             },
-            "./": {
+            "./*": {
```

Comment:
I'm not entirely sure if this change is valid - from what I understand it is because this is about the **exact** match and inexact matches are defined in the `patternProperties`, right?